---
string[] name
iai_bullet_sim/MultiBody[] body
uint8[] controller
string[] body_name